Tools/Scripts/delete-if-too-large

```diff
@@ -41,14 +41,18 @@ def main():
             print(f'{directory} is not a directory')
             return 1
 
-        tenGigabytes = 1024 * 1024 * 1024 * 10
-        directorySize = sum(file.stat().st_size for file in directoryPath.rglob('*'))
-        if directorySize >= tenGigabytes:
-            os.remove(directory)
+        tenGigabytesInKiloBytes = 1024 * 1024  * 10
+        
+        #directorySize will be in kilobytes
+        directorySize = int(subprocess.check_output(['sudo', 'du', '-sk', directoryPath]).decode().strip().split()[0])
+
+        if directorySize >= tenGigabytesInKiloBytes:
+            print(f'{directory} is {directorySize}. Deleting') 
```

Comment:
This log line won't read well when printed, because it lacks units for size (and doesn't even explain that the number is a size), and lacks a period at the end.